.github/workflows/release-notifications.yml

```
@@ -0,0 +1,21 @@
+name: Release Notifications
+
+on:
+  release:
+    types: [published]
```

I don't think we'll want to run this for all our prereleases, only on stable releases. [Docs reference](https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#webhook-payload-object-42)

```suggestion
    types: [released]
```